webpack.prod.config.ts

```diff
@@ -16,11 +16,11 @@
  * @fileoverview Production environment config file for Webpack.
  */
 
-const { merge } = require('webpack-merge');
-const common = require('./webpack.common.config.ts');
-const path = require('path');
-const webpack = require('webpack');
-const analyticsConstants = require('./assets/analytics-constants.json');
+var { merge } = require('webpack-merge');
+var common = require('./webpack.common.config.ts');
+var path = require('path');
+var webpack = require('webpack');
+var analyticsConstants = require('./assets/analytics-constants.json');
```

Comment:
Why are these changed?

Reply:
Ts start complaning about redeclaring path and others changed here. I think it will be fixed with ts and angular upgrade but until then we are stuck with this.